src/sentry/feedback/blueprints/api.md

```diff
@@ -84,7 +85,7 @@ Retrieve a collection of feedback items.
 | timestamp         | string           | ISO-8061 formatted UTC datetime.                    |
 | url               | string           | The page the feedback was triggered on.             |
 | user              | object           | The authorized user's information.                  |
-| user.display_name | optional[string] | -                                                   |
```

Comment:
should the api be updated to conform to the blueprint?

Reply:
I believe the backend already has this change, which is why I changed the blueprint

Reply:
<img width="373" alt="SCR-20231013-jmci" src="https://github.com/getsentry/sentry/assets/56095982/bb68e553-6d78-40f2-baa4-eda8541e6a8d">
this is the object being returned

Reply:
which isn't exactly updating the blueprint either, we're missing `isStaff` field and we have an extra `username` field but not sure if we should change that now 🤔

Reply:
i meant should the backend be returning `display_name` instead of `name`

Reply:
cc @billy are we passing through the `sdk.setUser()` call into this field? Does the shape if it (ie: isStaff, display_name, etc) depend on what people are passing into the sdk?

Reply:
not sure if that's something we can do because the user object is based on the event type

Reply:
Yes these user fields are user-configurable, though we suggest certain fields

Reply:
@billyvg should we make changes to the blueprint to reflect what the BE is returning? e.g. add `isStaff`, remove `username`, rename `display_name` to `name`? or change the BE to reflect the blueprint as it currently is

Reply:
@michellewzhang since all of those are user-configurable, we should leave the schema as open. not sure if blueprint has an option for additional properties. 

Also, W/ us using the issue platform imminently, we'll end up switching away from this schema, so i wouldn't worry about this schema until we get that figured out.